CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,10 @@
+# Microsoft Open Source Code of Conduct
```

Comment:
@alexneyman-MSFT I believe these changes went in already. Why are they showing up as part of your change